dev-resources/less/pages/tyomaapaivakirja.less

```diff
@@ -0,0 +1,107 @@
+.paivakirja-toimitus {
+    display: flex;
+    align-items: center;
+
+    .kohta {
+        padding-left: 10px;
+    }
+
+    .pallura {
+        height: 8px;
+        width: 8px;
+        left: 2px;
+        top: 2px;
+        border-radius: 10px;
+    }
+    .ok {
+        background: @green-default;
+    }
+    .myohassa {
+        background: @lemon-default;
+    }
+    .puuttuu {
+        background: @red-dark;
+    }
+}
+
+.puuttuu-tausta {
+    background: @lemon-light;
+}
+
+.tyopaiva {
+    display: flex;
+
+    :first-child {
+        width: 20px;
+        padding-left: 5px;
+    }
+}
+
+.tyomaa-haku-suodatin {
+    width: 50%;
+}
+
+.nakyma-otsikko-tiedot {
+    display: flex;
+    flex-direction: row;
+    color: @black-light;
+
+    > * {
+        margin-top: 10px;
+        margin-left: 20px;
+    }
+
+    :first-child {
+        margin-left: 0px;
+    }
+}
+
+.flex-gridit {
+    display: flex;
+    flex-direction: row;
+    gap: 40px;
+
+    .nakyma-otsikko {
+        background: @harmaa-vaalea;
+    }
+
+    // Meillä ei käytetä ollenkaan valkoista taustaa?
+    // Toisin kun figmassa sanotaan
+    // Käytetäänkö @harjan-taustavari ?
+    .nakyma-valkoinen-solu {
+        background: #FFFFFF;
```

Comment:
Hyvä huomio, taustaväri pitäisi tarkistaa. Jos pidetään valkoinen, käytä mieluummin @white-default, jotta värit on yhtenäisesti kirjoitetu

Reply:
Käytetään @white-default kun tämä on näin speksattu.